php/libraries/Candidate.class.inc

```diff
@@ -630,43 +630,17 @@ class Candidate
     {
         $factory = NDB_Factory::singleton();
         $db      = $factory->database();
-
         $query  = "SELECT MAX(s.VisitNo)+1
                     FROM candidate AS c
                         LEFT JOIN session AS s USING (CandID)
                     WHERE c.CandID=:CaID AND s.Active='Y' AND c.Active='Y'";
         $result = $db->pselectOne($query, array('CaID' => $this->getCandID()));
-
         if ($result === null) {
             return 1;
         }
         return intval($result);
     }
 
-    /**
-     * Returns list of consents and their respective statuses for this candidate
-     *
-     * @return array List of consents and their associated values for this candidate
-     *               The keys of the arrays are the IDs of the consents
-     */
-    public function getConsents(): array
```

Comment:
why is the candidate class being touched and why is this entire function getting deleted? I'm pretty sure it's being used somewhere

Reply:
@zaliqarosli thanks, not sure how that happened.